bin/dfx-ii-init-salt

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+SOURCE_DIR="$(dirname "$(realpath "${BASH_SOURCE[0]}")")"
+
+print_help() {
+  cat <<-EOF
+
+  Initialize the randomness in Internet Identity.
+  Otherwise this will be done the first time a user signs in and this will fail
+  if done multiple times simultaneously.
+	EOF
```

Comment:
The herefile should be indented with tabs. The `<<-EOF` will remove the tab indentation.  As is, the actual text printout will be indented with two spaces.  See: https://tldp.org/LDP/abs/html/here-docs.html#:~:text=The%20%2D%20option%20to%20mark%20a%20here%20document%20limit%20string%20(%3C%3C%2DLimitString)%20suppresses%20leading%20tabs%20(but%20not%20spaces)%20in%20the%20output.%20This%20may%20be%20useful%20in%20making%20a%20script%20more%20readable.

Reply:
Done